samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/PublishedAssessmentFacadeQueries.java

```diff
@@ -1430,30 +1432,58 @@ public List<PublishedAssessmentFacade> getBasicInfoOfAllPublishedAssessments2(St
 
 		List<PublishedAssessmentFacade> pubList = new ArrayList<>();
 		Map groupsForSite = null;
-		Map releaseToGroups;
+		Map releaseToGroups = new HashMap();
 		String lastModifiedBy = "";
 		AgentFacade agent = null;
-
+		Long assessmentId;
```

Comment:
```suggestion
		Long assessmentId = p.getPublishedAssessmentId();
```

Reply:
p is defined on the loop
https://github.com/sakaiproject/sakai/pull/12713/files#diff-e7d70308680f6b42bafb2e164b377af7182711a351475f9bcf4e10bcee82c354R1450